docs/content/latest/contribute/core-database/build-from-src-centos.md

```diff
@@ -53,7 +53,7 @@ Update packages on your system, install development tools and additional package
 ```sh
 sudo yum update
 sudo yum groupinstall -y 'Development Tools'
-sudo yum install -y ruby perl-Digest epel-release ccache git python2-pip python-devel python3 python3-pip python3-devel
+sudo yum install -y sudo ruby perl-Digest epel-release ccache git python2-pip python-devel python3 python3-pip python3-devel which
```

Comment:
So how are you running this command if you don't have sudo installed? 🙃

Reply:
OK, removed.